flowzone.yml

```diff
@@ -1919,6 +1929,10 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJSON(needs.is_docker.outputs.docker_test_matrix) }}
 
+    # https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings
+    permissions:
+     id-token: write
```

Comment:
> You may need to specify additional permissions here, depending on your workflow's requirements.

I think this removes almost all permissions from the GITHUB_TOKEN secret we are passing into docker bake as a build secret. Since we don't know how that build arg is used we should try to keep the permissions the same and just add what we need.